django_netjsongraph/static/netjsongraph/js/topology-switcher.js

```diff
@@ -1,17 +1,17 @@
 $(function() {
-    $('#datepicker').datepicker();
+    $('#dp').datepicker();
     d = new Date();
     month = d.getMonth()+1;
     day = d.getDate();
     default_date = (month<10? '0':'') + month + '/' +
         (day<10? '0':'') + day + '/' +
         d.getFullYear();
-    $('#datepicker').val(default_date);
+    $('#dp').val(default_date);
     current_date = d.getFullYear() + '-' +
         (month<10? '0':'') + month + '-' +
         (day<10? '0':'') + day;
     $('#submit').click(function() {
```

Comment:
instead of the submit event, can you use the change event of the date input? It would be more usable